src/project-tests/product-landing-page-tests.js

```diff
@@ -121,10 +121,19 @@ export default function createProductLandingPageTests() {
           video.nodeName === 'VIDEO' || video.nodeName === 'IFRAME',
           '#video should be an <iframe> or <video> element '
         );
+        // To accommodate `<source>` elements within `<video>` elements
+        // https://developer.mozilla.org/en-US/docs/Web/HTML/Element/source
+        const sourceNode = video.hasChildNodes();
+        let videoSrc;
+        if (sourceNode) {
+          videoSrc = video.children[0].hasAttribute('src');
+        } else {
+          videoSrc = video.hasAttribute('src');
```

Comment:
It might be safer to make this the primary check? If the `<video>` element does have a `src` attribute, then you can skip any further checking?